content/_components/onedrive/index.md

```diff
@@ -6,152 +6,158 @@ description: This is the component for working with Microsoft OneDrive storage s
-* **Expand Children** - Checkbox, trigger retrieves files from subfolders of chosen path, if enabled. Disabled by default.
-* **Enable File Attachments** - Checkbox for attaching files content to response. Disabled by default.
 
-## Actions
-
-### Get File
+#### Limitations
 
-Action to get item from OneDrive by provided path in selected disc.
+* OneDrive API doesn't support filtering - as result we collect information about all files from selected folders and filter them locally (inside component) for each trigger execution
+* From point above, option `Emit page` not always emit records according to `Page Size`
+* `Include subfolders` increase amount of API calls - additional call for each subfolder and inner folders
 
-![Get File](img/get-file.png)
-
-#### Input fields description
+## Actions
 
-* **Drive Identity** - OneDrive instance to work with. Selects by owner
-* **Enable File Attachments** - checkbox for attaching files content to action response
+### Create Folder
 
-#### Metadata fields description
+Create new folder in provided `path`. If `path` not exist component will fail.
 
-* **Path** - Full path to item to create or replace
+#### Configuration Fields
 
-#### Input example:
+* **Drive Identity** - (dropdown, required): OneDrive instance to work with
+* **Conflict Behavior** - (dropdown, optional, `Fail` by default) - Select one of options to handle case when folder already exists:
+  * **Fail** - Fails if folder with same name already exists under provided `path`
+  * **Generate new name** - If folder with same name already exists under provided `path` then will be created new folder with different name. Examples: `folder_name` (already exist) -> `folder_name 1` (will be created), `folder_name 1` (already exist) -> `folder_name 1 1` (will be created)
+  * **Upsert** - If folder already exist, you will get information about this folder
 
-```
-{
-    "path": "base_folder/inner_folder/file.any"
-}
-```
+#### Input Metadata
 
-### Upsert File
+* **Path** - (string, required): Full path to folder where you need to create new folder, ex: `Monthly reports/November`
+* **Name** - (string, required): Name of new folder
 
-Action upserts (create or replace) with first file from attachment by provided path in Microsoft One Drive
+#### Output Metadata
 
-![Upsert File](img/upsert-file.png)
+Metadata of created folder
 
-#### Input fields description
+### Delete File
 
-* **Drive Identity** - OneDrive instance to work with. Selects by owner
+Action to delete item from OneDrive by provided path in selected disc.
 
-#### Metadata fields description
+#### Configuration Fields
 
-* **Path** - Full path to item to create or replace
+* **Drive Identity** - (dropdown, required): OneDrive instance to work with
+* **Emit strategy when file not found** - (dropdown, optional, `Emit nothing` by default) - select one of options to handle case when file not exist:
```

Comment:
* **Emit strategy when file not found** - (dropdown, optional, `Emit nothing` by default) - select one of the options to handle the case when the file does not exist: